js/render/painter.js

```diff
@@ -27,7 +27,7 @@ function Painter(gl, transform) {
 
     // Within each layer there are 3 distinct z-planes that can be drawn to.
     // This is implemented using the WebGL depth buffer.
-    this.numSublayers = 3;
+    this.numSublayers = 14;
```

Comment:
Maybe safest to not mention a specific number in that comment. It could easily come out of sync again.